aospy/calc.py

```diff
@@ -374,9 +375,11 @@ def _add_grid_attributes(self, ds, n=0):
                 except ValueError:
                     ds = ds
                 ds = ds.set_coords(name_int)
-                if not ds[name_int].equals(model_attr):
-                    warnings.warn("Model coordinates for '{}' "
-                                  "do not match those in Run".format(name_int))
+                if not np.array_equal(ds[name_int], model_attr):
```

Comment:
Sorry, I really shouldn't be doing this commit by commit... I see what's happening here now.